libraries/adaptive-expressions/src/builtinFunctions/stringTransformEvaluator.ts

```diff
@@ -19,10 +20,10 @@ export class StringTransformEvaluator extends ExpressionEvaluator {
      * @param type Name of the built-in function.
      * @param func The string transformation function, it takes a list of objects and returns an string.
      */
-    public constructor(type: string, func: (arg0: any[]) => string) {
+    public constructor(type: string, func: (arg0: any[], options: Options) => ValueWithError) {
```

Comment:
Is this an internal class? This could be considered a public breaking change.

Reply:
I double checked this class. It is exported to external access. So I reverted the change of the interface.